plugins/git/plugin.zsh

```diff
@@ -95,7 +95,24 @@ prompt_geometry_git_remote_check() {
 }
 
 prompt_geometry_git_symbol() {
-  echo "$(prompt_geometry_git_rebase_check) $(prompt_geometry_git_remote_check)"
+  local git_rebase="$(prompt_geometry_git_rebase_check)"
+  local git_remote="$(prompt_geometry_git_remote_check)"
+  local render=""
+
+  if [[ $git_rebase != "" ]]; then
```

Comment:
I'd rather use `-n` instead of `!=`:

    if [[ -n $git_rebase ]]; then

[Here](http://zsh.sourceforge.net/Doc/Release/Conditional-Expressions.html) is the list of conditionals available.